specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,711 @@
+            },
+            "tenantId": {
+              "type": "string",
+              "description": "Provider's tenant id."
+            }
+          }
+        },
+        "managedApplication": {
+          "type": "object",
+          "description": "Provider's Managed-Application info",
+          "properties": {
+            "publisherId": {
+              "type": "string",
+              "description": "Provider's publisher id"
+            },
+            "offerId": {
+              "type": "string",
+              "description": "Provider's offer id"
+            }
+          }
+        },
+        "targets": {
+          "type": "array",
+          "description": "The list of targets available from this provider",
+          "items": {
+            "$ref": "#/definitions/TargetDescription"
+          }
+        },
+        "skus": {
+          "type": "array",
+          "description": "The list of skus selected for this provider",
+          "items": {
+            "$ref": "#/definitions/SkuDescription"
+          }
+        }
+      }
+    },
+    "TargetDescription": {
+      "type": "object",
+      "description": "Descript provider's target information to end customer.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique target id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Display name of target"
+        },
+        "description": {
+          "type": "string",
+          "description": "Description about this sku"
+        },
+        "acceptedDataFormats": {
+          "type": "array",
+          "description": "List of data formats",
+          "items": {
+            "type": "string"
+          }
+        },
+        "acceptedContentEncodings": {
+          "type": "array",
+          "description": "List of content encodings",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SkuDescription": {
+      "type": "object",
+      "description": "Descript provider's information to end customer.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "unique sku id"
+        },
+        "name": {
+          "type": "string",
+          "description": "display name of sku"
+        },
+        "description": {
+          "type": "string",
+          "description": "Description about this sku"
+        },
+        "targets": {
+          "type": "array",
+          "description": "list of targets for the sku",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Provider": {
+      "type": "object",
+      "description": "Json representation of provider, used for json serialization and deserialization and save the meta information of provider.",
+      "properties": {
+        "providerId": {
+          "type": "string",
+          "description": "partner id field"
```

Comment:
These descriptions are not very good. 

saying `"XXX field"` is not a good way to document an API.